src/differential_privacy/dp_parameters.rs

```diff
@@ -22,19 +24,21 @@ impl DpParameters {
         tau_thresholding_share: f64,
         privacy_unit_max_multiplicity: f64,
         privacy_unit_max_multiplicity_share: f64,
+        max_user_groups: u64,
     ) -> DpParameters {
         DpParameters {
             epsilon,
             delta,
             tau_thresholding_share,
             privacy_unit_max_multiplicity,
             privacy_unit_max_multiplicity_share,
+            max_user_groups,
```

Comment:
There is no notion of user in qrlew -> please align the naming e.g. `max_group_per_privacy_unit` or `max_privacy_unit_groups`